hack/generated/config/manager/manager_image_patch.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     spec:
       containers:
         # Change the value of image field below to your controller image URL
-        - image: localhost:5000/azure-service-operator:latest
+        - image: localhost:5000/azureserviceoperator:latest
```

Comment:
Do we need to somehow disambiguate this from the docker image published for v1?

Reply:
Or will `v1` just always be `azureserviceoperator:1.x` and `v2` is always `azureserviceoperator:2.x`? If so that make sense.

Reply:
@matthchr, yes exactly. And we won’t push v2 to `:latest` until v2 is complete.

Reply:
`:latest` is basically "useless" in our use case anyway I think, since pin releases to a particular version even for ASOv1 so that people don't accidentally get updates when we push new container images.

With that said, what you're saying makes sense. Looks like a good plan to me!